html-css-js/html-js/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="css/style.css" rel="stylesheet">
+  </head>
+
+  <button onClick="clica">CLICA</button>
+  <body>
```

Comment:
você deve adicionar um elemento div com uma classe para receber o resultado do evento do botão
```suggestion
  <body>
  <div class="resultado"></div>
```